server/views/partials/toCourtLink.njk

```diff
@@ -0,0 +1,17 @@
+{# Macro to format the CVP link based on known information and the feature switch value #}
+
+{% macro toCourtLink(item, feature) %}
+    {% if feature and item.hmctsNumber %}
+      <a class="govuk-link govuk-link--no-visited-state govuk-!-text-break-word" href="https://HMCTS{{ item.hmctsNumber }}@meet.video.justice.gov.uk" rel="noreferrer noopener" target="_blank">
```

Comment:
I suspect we'll have a another round of small change to format the actual links to get us into meetings. Should eventually look something like: ` https://join.meet.video.justice.gov.uk/HMCTS/#/?conference=hmcts2189@meet.video.justice.gov.uk&pin=9658`

Reply:
Indeed.